lib/icinga/clusterevents-check.cpp

```diff
@@ -121,8 +200,32 @@ void ClusterEvents::ExecuteCheckFromQueue(const MessageOrigin::Ptr& origin, cons
 		CheckResult::Ptr cr = new CheckResult();
 		cr->SetState(ServiceUnknown);
 		cr->SetOutput("Endpoint '" + Endpoint::GetLocalEndpoint()->GetName() + "' does not accept commands.");
-		Dictionary::Ptr message = MakeCheckResultMessage(host, cr);
-		listener->SyncSendMessage(sourceEndpoint, message);
+
+		if (params->Contains("source")) {
+			Dictionary::Ptr executedParams = new Dictionary();
+			executedParams->Set("execution", params->Get("source"));
+			executedParams->Set("host", params->Get("host"));
+			if (params->Contains("service"))
+				executedParams->Set("service", params->Get("service"));
+			double now = Utility::GetTime();
+			executedParams->Set("error", cr->GetOutput());
```

Comment:
If `!params->Contains("source")`, we submit the error as check result.

Submit the error as command execution result.

Reply:
Sorry, to avoid misunderstandings, could you elaborate on this?
I would interpret this as adapting the `else {` of this, block, but the line you put this comment does not compute. 

Thanks!

Reply:
No, I mean... just forget the first sentence of mine.